screens/TimerScreen.js

```diff
@@ -58,6 +59,12 @@ const TimerScreen = ({ navigation }) => {
         interruptionModeAndroid: INTERRUPTION_MODE_ANDROID_DO_NOT_MIX,
         shouldDuckAndroid: false,})}
 
+    const handleBack = () => {
+        ScreenOrientation.unlockAsync(ScreenOrientation.OrientationLock.PORTRAIT);
+        navigation.navigate("Home")
+        return;
+        }
+
     const renderTime = ({ remainingTime }) => {
         if (remainingTime === 0) {
           return <Text style={{ fontSize: '80em' }} className="text-center font-bold">Drink!</Text>;
```

Comment:
tailwind, only reason to ever use the styles class is for the AndroidSafeArea really cause we can't do that easily with tailwind

Reply:
you can do text-[80em] if none of the other presets work for it too